go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/containerd/go-runc v0.0.0-20201020171139-16b287bc67d0 // indirect
 	github.com/containerd/ttrpc v1.0.2 // indirect
 	github.com/containerd/typeurl v1.0.1 // indirect
-	github.com/docker/cli v20.10.0-beta1.0.20201103165149-c20be83d6b34+incompatible
```

Comment:
I think this should be revisited :thinking:

Reply:
Updated to 

```
	github.com/docker/cli v20.10.0+incompatible
	github.com/docker/docker v20.10.0+incompatible
```